### PR TITLE
feat(track order status): Have a buyer be able to track his/her order…

### PR DESCRIPTION
**Pull request by robsdagreat**
… status

What does this PR do?
Have a buyer be able to track his order status in real time

Description of Task to be completed?
Having a buyer be able to track his/her order status throughout the transaction stages
Having a web socket on the backend to notify the frontend whenever there is an updated  order status

What are the relevant pivotal tracker stories

#187570887

### CODE REVIEW

File: .husky/_/pre-commit [hunk 1]
```diff
@@ -1,9 +1,2 @@
-#!/bin/sh
-
-# Run linting and formatting
-npm run lint:fix
-npm run format
-
-
-# If everything passes, allow the commit
-# If any of the commands fail, the commit will be aborted
+#!/usr/bin/env sh
```
Comment (kirengamartial) [inline .husky/_/pre-commit:1]: why did you change this file it is needed

**Review by kirengamartial — CHANGES_REQUESTED** [review]
consider the requested changes

File: .husky/_/pre-commit (hunk 1 above)
Comment (robsdagreat) [reply .husky/_/pre-commit:1]: Actually that's because of the rebasing I did , but literally I didn't need to change that one.

**Review by Balinda21 — APPROVED** [review]
Well done @robsdagreat.
The codes looks clean and ready for merge.

File: src/utilis/orderStatusConstants.ts
Comment (Ishimwe7) [inline src/utilis/orderStatusConstants.ts:1]: Here you can also add a status for paid orders

**Review by Ishimwe7 — CHANGES_REQUESTED** [review]
Your code looks clean, in the meantime please consider feedbacks

File: src/utilis/orderStatusConstants.ts
Comment (robsdagreat) [reply src/utilis/orderStatusConstants.ts:1]: Good Idea @Ishimwe7 . I have already it as you required

**Review by DeliceLydia — APPROVED** [review]
